package.json

```diff
@@ -25,6 +25,8 @@
     "antd": "^5.21.2",
     "autoprefixer": "^10.4.20",
     "classnames": "^2.5.1",
+    "formik": "^2.4.6",
+    "formik-antd": "^3.0.0-beta.9",
```

Comment:
Is there a specific reason we need to use a beta version of `formik-antd`?

Reply:
Ohh gotya that makes sense. Looks like based on the issue there hasn't been a ton of motion so maybe not something we have to worry about much https://github.com/jannikbuschke/formik-antd/issues/204